osgp/platform/osgp-adapter-kafka-distributionautomation/src/main/java/org/opensmartgridplatform/adapter/kafka/da/application/config/KafkaConfig.java

```diff
@@ -0,0 +1,102 @@
+package org.opensmartgridplatform.adapter.kafka.da.application.config;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import javax.annotation.Resource;
+
+import org.opensmartgridplatform.adapter.kafka.MeterReading;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.core.env.Environment;
+import org.springframework.kafka.annotation.EnableKafka;
+// TODO - Enable when consumer is implemented
+//import org.springframework.kafka.config.ConcurrentKafkaListenerContainerFactory;
+//import org.springframework.kafka.core.ConsumerFactory;
+//import org.springframework.kafka.core.DefaultKafkaConsumerFactory;
+import org.springframework.kafka.core.DefaultKafkaProducerFactory;
+import org.springframework.kafka.core.KafkaTemplate;
+
+@EnableKafka
+@Configuration
+public class KafkaConfig {
+    @Resource
+    private Environment environment;
+
+    @Value("${distributionautomation.kafka.producer.topic}")
+    private String topicProducer;
+
+    // TODO - Enable when consumer is implemented
+    // @Value("${distributionautomation.kafka.consumer.concurrency}")
+    // private Integer concurrency;
+    //
+    // @Value("${distributionautomation.kafka.consumer.poll.timeout}")
+    // private Integer pollTimeout;
+
+    private Map<String, Object> producerConfigs() {
+        final Map<String, Object> properties = this.createCommonProperties();
+        properties.put("key.serializer", "org.apache.kafka.common.serialization.StringSerializer");
+        properties.put("value.serializer",
+                "org.opensmartgridplatform.adapter.kafka.da.infra.kafka.out.MeterReadingSerializer");
+        KafkaProperties.producerProperties()
+                .forEach((k, v) -> this.addIfExist(properties, k, "distributionautomation.kafka.producer", v));
+        return properties;
+    }
+
+    private Map<String, Object> createCommonProperties() {
+        final Map<String, Object> properties = new HashMap<>();
+        KafkaProperties.commonProperties()
+                .forEach((k, v) -> this.addIfExist(properties, k, "distributionautomation.kafka", v));
+
+        return properties;
+    }
+
+    // TODO - Enable when consumer is implemented
+    // @formatter:off
+//    @Bean
+//    public Map<String, Object> consumerConfigs() {
+//        final Map<String, Object> properties = this.createCommonProperties();
+//
+//        properties.put("key.deserializer", "org.apache.kafka.common.serialization.StringDeserializer");
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://ci.opensmartgridplatform.org/sonarqube/coding_rules#rule_key=squid%3ACommentedOutCodeLine)